src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.PackTool.targets

```diff
@@ -49,7 +49,7 @@ Copyright (c) .NET Foundation. All rights reserved.
         <PackagePath>tools/$(targetframework)/any/%(_GeneratedFiles.RecursiveDir)%(_GeneratedFiles.Filename)%(_GeneratedFiles.Extension)</PackagePath>
       </TfmSpecificPackageFile>
 
-      <TfmSpecificPackageFile Include="@(ResolvedFileToPublish->'$(PublishDir)%(RelativePath)')">
+      <TfmSpecificPackageFile Include="@(ResolvedFileToPublish->'$([System.IO.Path]::GetFullPath($(PublishDir)))%(RelativePath)')">
```

Comment:
I believe there's an NormalizePath intrinsic now that is preferable to use, but I'm not sure what version of msbuild added that and if we can depend on it. cc @jeffkl

Reply:
I talked to Mihai, the difference is on Windows it is faster and on unix it can replace \ to /. So NormalizePath it is the way to go.